utils.py

```diff
@@ -16,6 +16,136 @@ def _get_random_age(rng):
 		age = round(float(draw))
 	return age
 
+# &sex
+def _get_random_sex():
+	p = np.random.rand()
```

Comment:
using `np.random` will disturb the source of randomness. we should `rng` everywhere to have control on the source of randomness.

Reply:
oops, will fix!